Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -23,6 +29,12 @@
     sprite: Clothing/Mask/gassecurity.rsi
   - type: Clothing
     sprite: Clothing/Mask/gassecurity.rsi
+    toggleAction:
```

Comment:
Could probably use a base prototype for masks that have ingestion blocker, etc to inherit from rather than redefining all of it here. for each prototype.